test/cases/ostree-raw-image.sh

```diff
@@ -933,9 +1036,43 @@ greenprint "fix stdio file non-blocking issue"
 sudo /usr/libexec/osbuild-composer-test/ansible-blocking-io.py
 
 # Test IoT/Edge OS
-sudo ansible-playbook -v -i "${TEMPDIR}"/inventory -e image_type="${OSTREE_OSNAME}" -e edge_type=edge-raw-image -e ostree_commit="${UPGRADE_HASH}" -e sysroot_ro="$SYSROOT_RO" /usr/share/tests/osbuild-composer/ansible/check_ostree.yaml || RESULTS=0
+sudo ansible-playbook -v -i "${TEMPDIR}"/inventory \
+    -e image_type="${OSTREE_OSNAME}" \
+    -e edge_type=edge-raw-image \
+    -e ostree_commit="${UPGRADE_HASH}" \
+    -e sysroot_ro="$SYSROOT_RO" \
+    -e test_custom_dirs_files="$CUSTOM_DIRS_FILES" \
+    /usr/share/tests/osbuild-composer/ansible/check_ostree.yaml || RESULTS=0
 check_result
 
+# Test the same playbook with the user created by Ignition
+if [[ ${IGNITION} -eq 0 ]]; then
```

Comment:
This step is duplicated (please see line 992), and I suspect it caused CI test failures.

Reply:
Thanks for catching this. I may have messed up things while rebasing on top of `main`.